scripts/change_url

```diff
@@ -74,6 +83,7 @@ then
 
 	# Store file checksum for the new config file location
 	ynh_store_file_checksum --file="/etc/nginx/conf.d/$new_domain.d/$app.d/php.conf"
+	#ynh_store_file_checksum --file="/etc/nginx/conf.d/$new_domain.d/$app.d/nginx-code-error.conf"
```

Comment:
(same as above)
Per the CI test for change_url, I think we should reenable this. Or was it failing somehow?